lib/cube/buildCubeShape/Cube.js

```diff
@@ -36,7 +36,7 @@ class Cube {
     this.dimension({ predicate, object }).update({ predicate, object })
   }
 
-  toDataset () {
+  toDataset ({ shapeGraph } = { shapeGraph: undefined }) {
```

Comment:
```suggestion
  toDataset ({ shapeGraph } = { }) {
```

`undefined` is unnecessary